ocular/src/keyring.rs

```diff
@@ -0,0 +1,752 @@
+use bech32::{Bech32, ToBase32};
+use bip32::PrivateKey;
+use bip39::{Mnemonic, Seed};
```

Comment:
The `bip32` crate has `bip39` support built-in:

```suggestion
use bip32::{PrivateKey, Mnemonic, Seed};
```

Reply:
Updated.